nyxx_lavalink/lib/src/node/Node.dart

```diff
@@ -0,0 +1,263 @@
+part of nyxx_lavalink;
+
+/// Represents an active and running lavalink node
+class Node {
+  /// Node options, such as host, port, etc..
+  NodeOptions options;
+
+  /// A map with guild ids as keys and players as values
+  final Map<Snowflake, GuildPlayer> _players = {};
+
+  /// Returns a map with all the players the node currently has
+  UnmodifiableMapView<Snowflake, GuildPlayer> get players => UnmodifiableMapView(this._players);
+
+  /// Last stats received by this node
+  StatsEvent? stats;
```

Comment:
why is that field writeable?

Reply:
that field has to be writeable since `StatsEvent` is periodically received from lavalink, also the data of the event itself can't be modified as it's marked as final

Reply:
Then it should be private with public getter